README.md

```diff
@@ -51,15 +51,15 @@ Set the margin to 0 for each direct child of:
 
 ### #6
 
-If you inspect the `.left` and `.right` elements, you will see that they have `width: 50%;` and `float: left`. However, they are not sitting next to each other.
+If you inspect the `.left` and `.right` elements, you will see that they have `width: 50%;` and `float: left`. They are not, however, sitting next to each other.
 
 Fix this issue without modifying any of `.left` and `.right`'s `padding`, `border`, or `width`.
 
 > Commit
 
 ### #7
 
-When the page is fewer than 600px wide, the background of `.main` turns red.
+When the page is fewer than 600px wide, the background of `.main` turns red. The code responsible for this is located in the [`@media`](https://www.w3schools.com/cssref/css3_pr_mediaquery.asp) query at the bottom of the stylesheet.
 
```

Comment:
it's good to be tricky since it stresses the importance of attention to detail 📏 📐 🤓